content/docs/03-cluster-profiles/1-task-define-profile.md

```diff
@@ -115,6 +115,22 @@ In order to allow packs to be added multiple times in a profile, add the followi
 
 <br />   
 
+<InfoBox>
+By Default, Palette uses the packName-chartNames convention while installing Helm Charts. However, if the name goes beyond 64 characters for a custom name, use pack.releaseNameOverride in the layer.
```

Comment:
By default Palette uses Helm chart release name in the format `packName-chartName`. We've seen cases where having a lengthy release name causes other side effects. You can customize Helm chart releaseNames using the format below